src/AchievementService/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
don't you need anything else here? mb you should check another services or consult Maria